tests/requirements_test.txt

```diff
@@ -1,2 +1,3 @@
 pytest
+pytest-async
```

Comment:
do we need to include `pytest-asyncio` here? it looks like it's skipping these tests because i don't have it installed

![Screenshot 2024-03-14 at 1 39 28 PM](https://github.com/cleanlab/cleanlab-studio/assets/13542329/819f68ab-7cd8-4948-b12c-1d3b49ffdd99)

Reply:
installing that lib allowed those tests to run for me (and they passed)

Reply:
typo, fixed now